### ISSUE TITLE
[Feature Request] AMOLED Black Mode

### ISSUE THREAD

**Issue by tib-emma** [hugomg/DailyPill#7]
I just want to say I love the app, but the interface is only light mode. It's not a big issue, but an AMOLED black mode would be perfect :)

**Comment by hugomg** [comment]
Thanks for the suggestion. Are you talking about the Android 10 feature? https://developer.android.com/guide/topics/ui/look-and-feel/darktheme

**Comment by tib-emma** [comment]
Yes, exactly :) the app and the popup are light mode, which hurts my eyes since I use only dark mode apps. Plus I have an AMOLED screen which can suffer burn-in and battery loss from light themes.

**Comment by TPS** [comment]
_@tib-emma Have you tried [`Override force-dark`](https://www.androidauthority.com/android-q-force-dark-mode-984090/)? Note that the setting doesn't survive reboots._

**Comment by tib-emma** [comment]
Force dark mode doesn't cause the app to be AMOLED black (hex #000000). See attached image. I also have my phone perform a reboot nightly to help with performance, so this solution is very inconvenient.
![Screenshot_20200902-171412_Daily Pill](https://user-images.githubusercontent.com/58507086/92037422-bdb99480-ed3f-11ea-9663-265861b20c67.jpg)

**Comment by tib-emma** [comment]
Force dark mode doesn't cause the app to be AMOLED black (hex #000000). See attached image. I also have my phone perform a reboot nightly to help with performance, so this solution is very inconvenient.
![Screenshot_20200902-171412_Daily Pill](https://user-images.githubusercontent.com/58507086/92037422-bdb99480-ed3f-11ea-9663-265861b20c67.jpg)

**Comment by hugomg** [comment]
I implemented dark mode (non AMOLED) for the next version of Daily Pill. Could you help me test if it is working? You can get version 1.3.1 by signing up for beta testing on the [Play Store page for the app](https://play.google.com/store/apps/details?id=br.com.gualandi.dailypill) or you can get the APK from the [github releases page](https://github.com/hugomg/DailyPill/releases/tag/v1.3.1)

As for the AMOLED black thing, would it be possible to do something on your phone to make all dark theme use #000000 as a background color? It feels a bit strange to have to add this kind of functionality to Daily Pill in particular when it sounds like it would be more useful if you were able to change the system defaults.

**Comment by TPS** [comment]
> [Sign] up for beta testing on the [Play Store page for the app](https://play.google.com/store/apps/details?id=br.com.gualandi.dailypill)

On any browser that 1 can sign into their Google Play account, https://play.google.com/apps/testing/br.com.gualandi.dailypill will  allow to sign up for testing versions directly.

**Comment by hugomg** [comment]
Thanks! That link is much better (* ^ ω ^) 

Em 3 de setembro de 2020 21:44:40 BRT, TPS <notifications@github.com> escreveu:

**Comment by emmatebibyte** [comment]
> I implemented dark mode (non AMOLED) for the next version of Daily Pill. Could you help me test if it is working? You can get version 1.3.1 by signing up for beta testing on the [Play Store page for the app](https://play.google.com/store/apps/details?id=br.com.gualandi.dailypill) or you can get the APK from the [github releases page](https://github.com/hugomg/DailyPill/releases/tag/v1.3.1)
> 
> As for the AMOLED black thing, would it be possible to do something on your phone to make all dark theme use #000000 as a background color? It feels a bit strange to have to add this kind of functionality to Daily Pill in particular when it sounds like it would be more useful if you were able to change the system defaults.

I am unfortunately unaware of any settings that do this, but if you know of anything, please let me know!

**Comment by hugomg** [comment]
Googling a bit more, it really seems like the only way to add "super dark" mode is inside the app itself. Ugh, how inconvenient!

I don't know how this could be implemented but I'd be open to merging a pull request if someone else can figure out a way to do it.
Another thing that could be helpful is if someone could make a list of some other apps on F-Droid that implement an "AMOLED optimized" dark mode. The source code for those could be used as an example. 

By the way, were you able to try the current implementation of dark mode? I'm curious if it is really working...

**Comment by emmatebibyte** [comment]
> By the way, were you able to try the current implementation of dark mode? I'm curious if it is working...

Yes, I was, and it works perfectly! I'll post some good examples of AMOLED implementation here later today :)
